app/code/Magento/Inventory/Model/ReservationCleanupInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Inventory\Model;
+
+/**
+ * Extension point for reservation cleanup (Service Provider Interface - SPI).
+ * Provide own implementation of this interface if you would like to replace cleanup strategy.
+ *
+ * @api
+ */
+interface ReservationCleanupInterface
+{
+    /**
+     * Clean reservation table to prevent overloading.
```

Comment:
I would not expose that reservation table is used on the level of this service, because potentially it could be Redis queue

Don't introduce a leaky abstraction , as we need to be agnostic to the DAL at this level of abstraction